ipn/backend.go

```diff
@@ -82,6 +82,13 @@ type Options struct {
 	//    an initial overwrite of backend state with Prefs.
 	StateKey StateKey
 	Prefs    *Prefs
+	// RelaynodeConfigPath points to an old-style relaynode
```

Comment:
How about `LegacyConfigPath` for the field name?

// LegacyConfigPath optionally specifies the old-style relaynode relay.conf
// location. If both LegacyConfigPath and StateKey are specified and the requested state
// doesn't exist in the tailscaled location, tailscaled migrates the config from
// LegacyConfigPath.

?

Reply:
(notably the word "optionally")

Reply:
Done.